doc/contribute/committer_expectations.rst

```diff
@@ -0,0 +1,272 @@
+Meeting before escalating the PR yourself.
+
+.. _pr_technical_escalation:
+
+PR Technical Escalation
+=======================
+
+In cases where you object to change requests from reviewers, Zephyr defines the
+following escalation process for resolving technical disagreements.
+
+- Resolve in the PR among assignee, maintainers and reviewer.
+
+  - Assignee to act as moderator if applicable.
+
+- Optionally resolve in the next `Zephyr Dev Meeting`_ or `Architecture Working
+  Group`_ meeting with more Maintainers and project stakeholders.
+
+  - The involved parties and the Assignee to be present when
+    the (escalated) issue is discussed.
+
+- TSC: Assignees can escalate to the TSC voting members and get a binding
+  resolution in the TSC by adding the `tsc`_ label on the PR.
+
+- Assignee to ensure the resolution of the escalation is reflected in the PR
+  review.
+
+.. _#pr-help: https://discord.com/channels/720317445772017664/997527108844798012
+
+.. _dev-review: https://github.com/zephyrproject-rtos/zephyr/labels/dev-review
+
+.. _Zephyr Dev Meeting: https://github.com/zephyrproject-rtos/zephyr/wiki/Zephyr-Committee-and-Working-Group-Meetings#zephyr-dev-meeting
+
+.. _Architecture Project: https://github.com/zephyrproject-rtos/zephyr/projects/18
+
+.. _Architecture Working Group: https://github.com/zephyrproject-rtos/zephyr/wiki/Architecture-Working-Group
+
+.. _tsc: https://github.com/zephyrproject-rtos/zephyr/labels/tsc
+
+Reviewer Expectations
+#####################
+
+- Be respectful when commenting on PRs. Refer to the Zephyr `Code of Conduct`_
+  for more details.
+
+- The Zephyr Project recognizes that reviewers and maintainers have limited
+  bandwidth. Prioritize review requests in the following order:
+
+    #. PRs related to items in the `Zephyr Release Plan`_.
+    #. PRs you previously reviewed and requested blocking changes.
+    #. PRs assigned to you as the area maintainer.
+    #. All other PRs.
+
+- Try to provide feedback on the entire PR in one shot. This provides the author
+  an opportunity to address all comments in the next PR update.
+
+- Partial reviews are permitted, but the reviewer must add a comment indicating
+  what portion of the PR they reviewed. Examples of useful partial reviews
+  include:
+
+  - Domain specific reviews (e.g. Devicetree).
+  - Code style changes that impact the readability of the PR.
+  - Reviewing commits separately when the requested changes cascade into the
+    later commits.
+
+- Avoid increasing scope of the PR by requesting new features, especially when
+  there is a corresponding :ref:`RFC <rfcs>` associated with the PR. Instead add
+  your suggestions as a comment to the :ref:`RFC <rfcs>`. This also encourages
+  more collaboration as it is easier for multiple contributors to work on a
+  feature once the minimum implementation has merged.
+
+- Mark trivial, non-functional, requests as "Non-blocking" in your comment.
```

Comment:
```suggestion
- When using the "Request Changes" option, mark trivial, non-functional, requests as "Non-blocking" in your comment.
```

Otherwise it raises the bar too much for informal discussions, especially for less experienced persons already impressed by public discussions with well known experts.